README.adoc

```diff
@@ -39,33 +39,29 @@ Explore how to push Docker images to a custom registry and run them inside of co
 :inventory-api: http://[hostname]:[inventory-node-port]/inventory/systems
 
 
-// =================================================================================================
-// What is {kube}
-// =================================================================================================
-
-== Why use Kubernetes services in the cloud?
-
-Single node {kube} clusters such as Minikube and Docker Desktop are great for testing your 
-microservices locally, but ultimately you will want a system that can handle production workloads. 
-Various clouds offer solutions for running your workloads in a {kube} cluster. Using a 
-cloud-based infrastructure allows you to focus on developing your microservices rather than 
-worrying about details that are related to the servers you deploy them to. Using cloud allows
-you to easily scale your microservices and serve them in a high-availability setup.
-
 // =================================================================================================
 // Introduction
 // =================================================================================================
 
 == What you'll learn
 
 You will learn how to deploy two microservices in Open Liberty containers to a {kube}
-cluster on IBM Cloud. You will use Helm to deploy these microservices using
-Open Liberty Helm charts. Helm is a package manager for {kube}. It uses templates to
-generate {kube} yaml files and then deploys and manages them as releases in your cluster.
+cluster on IBM Cloud. 
+
+Kubernetes is an open source container orchestrator that automates many tasks involved in 
+deploying, managing, and scaling containerized applications. If you would like to learn
+more about Kubernetes, check out the https://openliberty.io/guides/kubernetes-intro.html[Deploying microservices to Kubernetes^]
+guide.
+
+Various clouds offer solutions for running your workloads in a {kube} cluster. Using a 
+cloud-based infrastructure allows you to focus on developing your microservices rather 
+than worrying about details that are related to the servers you deploy them to. Using 
+cloud allows you to easily scale your microservices and serve them in a high-availability 
+setup.
 
 IBM Cloud offers a hosted Kubernetes service that is called IBM Cloud Kubernetes Service (IKS). 
 IKS is a service that is offered as part of IBM's public cloud offerings. It provides a 
-hosted {kube} cluster that you can deploy your microservices to. You will use it 
+hosted {kube} cluster that you can deploy your microservices to. You will use IKS 
 with IBM Cloud Container Registry.
```

Comment:
I will move "You will use IKS with IBM Cloud Container Registry" to below.  Best to have this in the last paragraph where we talk about what you will do.

Reply:
Also, we should explain what is a container registry and why at an appropriate location in the guide.